src/styles/styles.scss

```diff
@@ -189,6 +189,11 @@ html {
     height: 18px;
     border-radius: 50%;
   }
+
+  .roundedLogo {
+    border-radius: 50%;
```

Comment:
should this also specify width and height?

Reply:
Each image specifies its own width and height within the components, and it differs from the classes above, so I can't reuse these. I wanted to have a global class only for rounding logos.